channels/in.m3u

```diff
@@ -247,6 +247,8 @@ http://103.199.161.254/Content/isaiaruvi/Live/Channel(IsaiAruvi)/index.m3u8
 http://103.199.161.254/Content/jaihind/Live/Channel(Jaihind)/index.m3u8
 #EXTINF:-1 tvg-id="JanTV.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Jan TV (360p) [Not 24/7]
 http://jantvstream.in:1935/edge1/sc1jantv.stream_aac/playlist.m3u8
+#EXTINF:-1 tvg-id="JanapriyamTV.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/rbn6YSb.png" group-title="Entertainment",Janapriyam TV
```

Comment:
Return "HTTP 451 Unavailable For Legal Reasons" if you try to run it outside of India.

```suggestion
#EXTINF:-1 tvg-id="JanapriyamTV.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/rbn6YSb.png" group-title="Entertainment",Janapriyam TV [Geo-blocked]
```